### ISSUE TITLE
Invisible black SVGs in white-on-black contrast mode

### ISSUE THREAD

**Issue by jpdevries** [jpdevries/eureka#96]
![](http://j4p.us/2b2S3D3T0y1Q/Screen%20Shot%202017-04-23%20at%202.13.02%20PM.png)

Should have a white bg
![](http://j4p.us/3m1d1I2D1J3H/Screen%20Shot%202017-04-23%20at%202.13.50%20PM.png)
